sdks/python/apache_beam/dataframe/convert_test.py

```diff
@@ -85,6 +85,94 @@ def test_convert(self):
       assert_that(pc_3a, equal_to(list(3 * a)), label='Check3a')
       assert_that(pc_ab, equal_to(list(a * b)), label='Checkab')
 
+  def test_convert_memoization(self):
```

Comment:
This test (and below) seem largely copies of the test above, but we don't even need to run the pipeline to test what this is testing. Perhaps it'd be better to limit it to what we're trying to test for clarity, i.e. the ids are the same.

Reply:
Done